applications/cluster_solver_check/cluster_solver_check.cpp

```diff
@@ -94,21 +94,26 @@ int main(int argc, char** argv) {
 
     // ED solver
     EdSolver ed_solver(parameters, dca_data_imag, dca_data_real);
-    ed_solver.initialize(0);
-    ed_solver.execute();
-    ed_solver.finalize(dca_loop_data);
+    if (!skip_ed) {
```

Comment:
So what's the purpose of skipping this?  Is this just a debug feature or important for the summit work?  Documentation is necessary.

Reply:
I used it during debugging of the new phase factor feature, not really useful for me anymore. I could either remove it, or document the usage in the header.

Reply:
Choose one, I'd prefer removal

Reply:
Removed in 60b9a4137cffd30f3cecf38e02c84ffd93dff07c